packages/docusaurus-theme-classic/src/theme/CodeBlock/index.js

```diff
@@ -90,7 +92,8 @@ export default ({children, className: languageClassName, metastring}) => {
       code={children.trim()}
       language={language}>
       {({className, style, tokens, getLineProps, getTokenProps}) => (
-        <pre className={classnames(className, styles.codeBlock)}>
+        <pre className={classnames(className, styles.codeBlock)} style={style}>
```

Comment:
> style={style}

Why is it needed here?

Reply:
1. The meaning of "style" is setting `background` and `textColor` by each prism theme automatically
The below one is without `style={style}`, I coundn't find the way to set `background` and `textColor` automatically without `style={style}`
![image](https://user-images.githubusercontent.com/29557494/75625304-7b2bc980-5bef-11ea-938c-c98f91deacfa.png)

2. And sorry I could set `style={style}` in filename div.
```jsx
{fileName && <div style={style} className={styles.fileName}>{fileName}</div>}
```

Reply:
Just use absolute position for this.

Reply:
Okay, so are we going to use boderBottom?

Reply:
Yes, it should turn out something like this:

![image](https://user-images.githubusercontent.com/4408379/75626892-60038e80-5bdc-11ea-96cd-5d0028ccb5f4.png)

Note that the copy button should be under the code title.

Reply:
okay! Ill work on it now.